src/ace/managers/sprite.c

```diff
@@ -23,14 +23,29 @@ typedef struct tSpriteChannel {
 
 static const tView *s_pView;
 static tSpriteChannel s_pChannelsData[HARDWARE_SPRITE_CHANNEL_COUNT];
-static const tHardwareSpriteHeader CHIP s_uBlankSprite;
+static ULONG *s_pBlankSprite;
+static UBYTE s_isOwningBlankSprite;
 static tCopBlock *s_pInitialClearCopBlock;
 
 static void spriteChannelRequestCopperUpdate(tSpriteChannel *pChannel) {
 	pChannel->ubCopperRegenCount = 2; // for front/back buffers in raw mode
 }
 
-void spriteManagerCreate(const tView *pView, UWORD uwRawCopPos) {
+void spriteManagerCreate(const tView *pView, UWORD uwRawCopPos, ULONG pBlankSprite[1]) {
+	if (pBlankSprite) {
+#ifdef ACE_DEBUG
+		if ((ULONG)pBlankSprite > memGetChipSize() - sizeof(ULONG)) {
+			logWrite("ERR: ILLEGAL NON-CHIP memory location for blank sprite!");
+		}
+#endif
+		s_isOwningBlankSprite = 0;
+		s_pBlankSprite = pBlankSprite;
+	} else {
+		s_isOwningBlankSprite = 1;
+		s_pBlankSprite = memAllocChipClear(sizeof(ULONG));
+		// Just to make sure we don't accidentally mismatch the control words size
+		_Static_assert(sizeof(ULONG) == sizeof(tHardwareSpriteHeader));
```

Comment:
can you by any chance check if static asserts are supported by Bebbo compiler? I don't have one at hand currently.

Reply:
I kicked off a Bebbo install and will see if it works

Reply:
Ok, Bebbo's gcc version did not like `_Static_assert` without a message, but now it's ok.